docs/storm-kafka-client.md

```diff
@@ -313,4 +313,37 @@ KafkaSpoutConfig<String, String> kafkaConf = KafkaSpoutConfig
   .setTupleTrackingEnforced(true)
 ```
 
-Note: This setting has no effect with AT_LEAST_ONCE processing guarantee, where tuple tracking is required and therefore always enabled.
\ No newline at end of file
+Note: This setting has no effect with AT_LEAST_ONCE processing guarantee, where tuple tracking is required and therefore always enabled.
+
+# Migrating a `storm-kafka` spout to use `storm-kafka-client`
```

Comment:
That's a good point. If users need to migrate a topology, they'll probably want to copy committed offsets as well. Maybe we should link to https://github.com/apache/storm/tree/master/external/storm-kafka-migration

Reply:
Good point @hmcl. Calling the heading "Translation from `storm-kafka` to `storm-kafka-client` spout properties".
@srdo I will submit a PR to link this table in the `storm-kafka-migration` README.

Reply:
@srishtyagrawal I meant the other way around, linking from here to the `storm-kafka-migration` README. Not sure why we'd want to link from `storm-kafka-migration` to here?

Reply:
@srdo I suggested the other way because the name `storm-kafka-migration` suggests that the  module will help with migrating storm-kafka, but either way is fine. I can add the link to `storm-kafka-migration` here.